samples/siwx91x_ota/sample.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+sample:
+  name: HTTP/HTTPS OTA
+  description: HTTP/HTTPS OTA firmware update application for SiWx917
+tests:
+  sample.net.http_otaf:
+    harness: net
+    platform_allow:
+      - siwx917_rb4338a
```

Comment:
This test also work with siwx917_rb4342a and dh2605a.
Maybe:

    filter: CONFIG_SOC_FAMILY_SILABS_SIWX91X